doc/modules/ROOT/pages/tutorials/index.adoc

```diff
@@ -0,0 +1,12 @@
+= Tutorials
```

Comment:
Just a comment on the name of this file; it makes it easier to find if we call it `tutorials.adoc` instead of the generic `index.adoc`

Reply:
I think it makes the url a bit nasty `.../tutorials/tutorials/...` but maybe that's ok

Reply:
oh it changes the URL as well? I was unaware

Reply:
At least I thought so, but now I'm not so sure haha. I guess we'll see :man_shrugging:

Reply:
you're right :( https://neo4j.com/docs/graph-data-science-client/1.6/tutorials/tutorials/